k8s/prod/common/em/em-npa.yaml

```
@@ -0,0 +1,29 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: em-npa
+  namespace: em
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.em-npa: glob:prod-*
+spec:
+  releaseName: em-npa
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-npa
+    version: 0.1.11
```

```suggestion
    version: 0.1.6
```
https://github.com/hmcts/rpa-native-pdf-annotator-app/blob/master/charts/em-npa/Chart.yaml#L3